src/bot_quake.py

```diff
@@ -1,13 +1,17 @@
-import unittest
-import os
+"""Il modulo urllib non ha un docstring."""
 from urllib.request import urlopen
 from datetime import date, timedelta
-from telegram.ext import Application, Updater, CommandHandler, MessageHandler, filters
 import ssl
+import os
+from telegram.ext import Application, CommandHandler, MessageHandler, filters
+from telegram import Update
+from telegram.ext import ContextTypes
+# import unittest
 
 
-# la funzione imposta i valori temporali dei dati da scaricare in un range che va dal giorno attuale indietro di un valore d_range
 def get_date_range(d_range):
+    """ la funzione imposta i valori temporali dei dati da scaricare
+    in un range che va dal giorno attuale indietro di un valore d_range"""
```

Comment:
Spostate gesta funziona in un nuovo file utils/helper

Reply:
Fatto!